src/python_ml_project_template/models/flow_predictor.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any
+
+import lightning as L
+import torch.nn.functional as F
+from torch import optim
+
+
+# TODO: flow predictor
+class FlowPredictorTrainingModule(L.LightningModule):
```

Comment:
ultimately, merger this file and the `artflownet.py` you wrote earlier. I *dont* think you'll need a separate `nets/` file at all b.c. you're importing from `rpad.pyg_utils`.